lib/src/wallet/keys/unified.rs

```diff
@@ -120,8 +120,9 @@ impl UnifiedSpendCapability {
             None
         };
         let transparent_receiver = if desired_receivers.transparent {
-            let key_index =
-                KeyIndex::hardened_from_normalize_index(self.addresses.len() as u32).unwrap();
+            let key_index = KeyIndex::from_index(self.addresses.len() as u32).unwrap();
+            //let key_index = KeyIndex::Normal(self.addresses.len() as u32);
```

Comment:
Shall we cut out this comment?

Reply:
I though I did! I think I did in a rebase that didn't end up being part of the stack. Will do!